crates/swc/tests/fixture/issues-1xxx/1660/case1/output/index.js

```diff
@@ -1,6 +1,6 @@
 var _class_call_check = require("@swc/helpers/_/_class_call_check");
 var _create_class = require("@swc/helpers/_/_create_class");
-console.log(/*#__PURE__*/ function() {
+console.log(/*#__PURE__*/ /*#__PURE__*/ function() {
```

Comment:
Here `#__PURE__` is printed twice, which may be due to an issue during codegen.

This issue did not occur before because the `take` method was used, ensuring that the comment is printed only once.